label_studio/webhooks/api.py

```diff
@@ -0,0 +1,125 @@
+import logging
+
+
+from django.utils.decorators import method_decorator
+from django_filters.rest_framework import DjangoFilterBackend
+import django_filters
+from drf_yasg.utils import swagger_auto_schema
+from drf_yasg import openapi
+from rest_framework import generics
+from rest_framework.decorators import action
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Webhook, WebhookAction
+from .serializers import WebhookSerializer, WebhookSerializerForUpdate
+from projects import models as project_models
+
+
+class WebhookFilterSet(django_filters.FilterSet):
+    project = django_filters.ModelChoiceFilter(
+        field_name='project', queryset=project_models.Project.objects.all(), null_label='isnull'
+    )
+
+
+@method_decorator(
+    name='get',
+    decorator=swagger_auto_schema(
+        tags=['Webhooks'],
+        operation_summary='List all webhooks',
+        operation_description="List all webhooks set up for your organization.",
+    ),
+)
+@method_decorator(
+    name='post',
+    decorator=swagger_auto_schema(
+        tags=['Webhooks'],
+        operation_summary='Create a webhook',
+        operation_description="Create a webhook for your organization.",
+    ),
+)
+class WebhookListAPI(generics.ListCreateAPIView):
+    queryset = Webhook.objects.all()
+    serializer_class = WebhookSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
Why don't we use `ViewClassPermission` here?

Reply:
because in OS we don't have Permission?